FirebaseAuth/Sources/Swift/Backend/VerifyClientRequest.swift

```diff
@@ -41,8 +41,8 @@ public class VerifyClientRequest: IdentityToolkitRequest, AuthRPCRequest {
   @objc public private(set) var isSandbox: Bool
 
   // TODO(ncooke3): This API should be unneccessary...
-  @objc override public init(endpoint: String,
-                             requestConfiguration: AuthRequestConfiguration) {
+  @objc public init(endpoint: String,
+                    requestConfiguration: AuthRequestConfiguration) {
```

Comment:
Wrt my comment above, is the idea to gradually remove these initializers in favor of a more complete one?

Reply:
Removed now. Thanks for pointing that out.